lib/logging.js

```diff
@@ -184,12 +212,15 @@
      * Variadic logging methods. The first argument is the log message, and any additional
      * arguments are passed along in the "args" array in the log event.
      */
-    Logger.prototype.debug   = makeLogMethod(exports.LOG_LEVEL_DEBUG);
-    Logger.prototype.info    = makeLogMethod(exports.LOG_LEVEL_INFO);
-    Logger.prototype.warn    = makeLogMethod(exports.LOG_LEVEL_WARNING);
-    Logger.prototype.error   = makeLogMethod(exports.LOG_LEVEL_ERROR);
-    Logger.prototype.warning = Logger.prototype.warn;
-    Logger.prototype.log     = Logger.prototype.info;
+    Logger.prototype.debug    = makeLogMethod(exports.LOG_LEVEL_DEBUG);
+    Logger.prototype.info     = makeLogMethod(exports.LOG_LEVEL_INFO);
+    Logger.prototype.warn     = makeLogMethod(exports.LOG_LEVEL_WARNING);
+    Logger.prototype.error    = makeLogMethod(exports.LOG_LEVEL_ERROR);
+    Logger.prototype.warning  = Logger.prototype.warn;
+    Logger.prototype.log      = Logger.prototype.info;
+    Logger.prototype.time     = makeTimerStartMethod();
+    Logger.prototype.timeEnd  = makeTimerEndMethod();
```

Comment:
I was using the time() and timeEnd() to measure performance, but I didn't want to leave all the log statements in the code.